ui/js/service/Startup.js

```diff
@@ -1,17 +1,21 @@
 import { bindActionCreators } from 'redux';
-import * as actionCreators from 'actions/counters';
-import WebSocketService from 'service/WebSocketService'
+import ReduxWebSocketService from '../websocket/ReduxWebSocketService'
+import { setHosts } from '../hosts/HostActions';
+const url = 'ws://localhost:5120/poll/';
 
 export default function startup(dispatch) {
-    const actions = bindActionCreators(actionCreators, dispatch);
-    /*
-    setInterval(() => {
-        console.log('Incrementing Foobar counter in background')
-        actions.increment('Foobar counter')
-    }, 3000)
-    */
-    const url = 'ws://localhost:5120/poll/'
-    const websocket = new WebSocketService(dispatch, url, 'poller');
-    websocket.startWebSocket()
+    new ReduxWebSocketService(url, 'hosts', dispatch)
+        .onOpened(({ws, id}) => {
+            ws.send(JSON.stringify({
+                id,
+                metricKey: '/hosts',
+                start: new Date(new Date - 3*3600*1000).getTime(),
+                step: 10000,
+                minSteps: 10,
+                tags: []
+            }));
+        })
+        .onMsg(({msg}) => dispatch(setHosts(msg.data)))
```

Comment:
Should we push this `dispatch` handling into `ReduxWebshopSocketService` ?
Such that it encapsulates dispatching into it and uses same reference for it as it uses to other type of messages? 
E.g. 
```javascript
onMsgEncapsulated: function(fn) {
   this.onMsg(({msg}) => dispatch(fn(msg))) 
}
```